packages/dcos-integration-test/extra/test_applications.py

```diff
@@ -59,6 +59,38 @@ def test_if_marathon_app_can_be_deployed_with_mesos_containerizer(cluster):
     cluster.marathon.deploy_test_app_and_check(app, test_uuid)
 
 
+def test_if_marathon_pods_can_be_deployed_with_mesos_containerizer(cluster):
+    """Marathon pods deployment integration test using the Mesos Containerizer
+
+    This test verifies that a Marathon pods can be deployed.
+    """
+
+    test_uuid = uuid.uuid4().hex
+
+    pod_definition = {
+        'id': '/integration-test-pods-{}'.format(test_uuid),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'environment': {'PING': 'PONG'},
+        'containers': [
+            {
+                'name': 'ct1',
```

Comment:
Nit: would be nice to use test_server here and pair it with something, so we can check service routing, but that can be a future PR so we get this smoke test in sooner